contrib/container/Dockerfile

```diff
@@ -128,6 +128,7 @@ COPY --from=prebuild /root/.local /root/.local
 
 # Copy source code
 COPY src/backend/InvenTree ${INVENTREE_BACKEND_DIR}/InvenTree
+COPY src/backend/requirements.txt ${INVENTREE_BACKEND_DIR}/requirements.txt
```

Comment:
Comment: Ok that is an embarrassing mistake, that is on me. Interesting that the CI did not fail

Reply:
Yes, it has been passing CI for quite some time now! I cannot work out how the failure during `invoke install` is ignored.

Reply:
That is interesting - is the raise not working?

Reply:
I only just added the raise